src/main/scala/com/microsoft/partnercatalyst/fortis/spark/CassandraTest.scala

```diff
@@ -35,7 +35,7 @@ object CassandraTest {
     val appName = this.getClass.getSimpleName
     val conf = new SparkConf()
       .setAppName(appName)
-      .set("spark.cassandra.connection.host", "52.177.191.215")
+      .set("spark.cassandra.connection.host", "40.79.72.103")
```

Comment:
Should this come from an environment variable so we don't have to re-compile the binary every time this changes? We already have `FORTIS_CASSANDRA_HOST` ([link](https://github.com/CatalystCode/project-fortis-pipeline/blob/32e8c1795a787db6ce4ae181e746d4d754f4ae19/ops/setup-environment.sh#L30)) so maybe we should just read this here?